### DISCUSSION TITLE
Spotify theme?

### DISCUSSION THREAD

**Discussion by Braeulias** [prasanthrangan/hyprdots#701] [Q&A] [answered]
Your Spotify looks so good and its matching to the theme, how did you do that? Is this part of hyprdots or is it something else (I got the newest version of hyprdots running). I am btw new to arch.

**Comment by mahaveergurjar** [comment, marked as answer]
The script uses Sleek theme and then uses the color scheme from the wallpaper. If this is not yet reflected on your system install spotify and spicetify-cli first.
YOu can also manually run the script name is restore_app.sh after install spotify and spicetify-cli 
The location of the script is Hyprdots/Scripts/.extra/ restore_app.sh

**Reply by Braeulias** [reply]
Thank you!

**Reply by Rafi1018** [reply]
So after doing install.sh & installing the Spotify & spicetify-cli I run the restore-app.sh right? (I am also interested on the steam theme provided so after installing steam I just gotta uncomment the lines before running the script right? )

**Reply by mahaveergurjar** [reply]
Right 👍

**Comment by mahaveergurjar** [comment]
If you understand this  answer then click on the answer and then  close this discussion

**Reply by Braeulias** [reply]
sorry I didnt now

**Reply by mahaveergurjar** [reply]
Ok no problem 👍

**Comment by kRHYME7** [comment]
Hey guys you can also run ``` Hyprdots man ``` to see some available commands for hyprdots
